hypervisor/dm/vpci/vpci_bridge.c

```diff
@@ -50,6 +50,38 @@
 #include <pci.h>
 #include "vpci_priv.h"
 
+/**
+ * @addtogroup vp-dm_vperipheral
+ *
+ * @{
+ */
+
+/**
+ * @file
+ * @brief Implementation of virtual pci bridge.
+ *
+ * This file defines operations to support virtual pci bridge. It implements struct pci_vdev_ops and related functions.
+ */
+
+/**
+ * @brief Initializes the vPCI bridge.
+ *
+ * A PCI bridge is also a PCI device. This function initializes the specified virtual PCI device as a PCI bridge. A vPCI
+ * bridge is based on a physical PCI bridge and is used for Service VM. It's usually used in the initialization phase of
+ * Service VM.
+ *
+ * It initializes all the PCI configuration space registers from the physical PCI bridge and emulates the type info.
+ * Note that the physical PCI bridge is already configured in config_pci_bridge().
```

Comment:
config_pci_bridge() is not an external interface, so, we can not refer it directly here.
We may say: `The physical PCI bridge registers are initialized and configured during the initialization of the physical PCI hierarchy.`